demos/dlgr/demos/chatroom/static/scripts/experiment.js

```diff
@@ -38,8 +12,14 @@ $(document).ready(function() {
     leave_chatroom();
   });
 
+  // Proceed to the waiting room.
```

Comment:
I think the canonical way to handle the waiting room redirect is now much simpler:
```
        // Proceed to the waiting room.
        $("#go-to-waiting-room").click(function() {
            dallinger.allowExit();  // if necessary... might not be
            dallinger.goToPage("waiting");
        });
```